src/lib/util/url.ts

```diff
@@ -6,6 +6,12 @@ export function getPathname() {
   return decodeURIComponent(location.pathname);
 }
 
+// remove the "dev:" prefix from the DeviceUID.
+export function getDeviceUID(deviceUID: string) {
+  return deviceUID.replace('dev:', '');
+}
+
 export function getNotehubEventsUrl(deviceUID: string) {
-  return `https://notehub.io/project/${APP_UID}/events?filter_dev=${deviceUID}`;
+  const deviceNumber = getDeviceUID(deviceUID);
```

Comment:
I’d add a comment. Future us might think this code is unnecessary, because the link to Notehub _seems_ to work if you include the `dev:` prefix, and you only notice the problem if you export events. I was confused until I figured that out.

Reply:
will do!